images/external_debug_protection.drawio

```diff
@@ -353,7 +353,7 @@
         <mxCell id="GbtCg3jwWPSh3iAjcVzU-3" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;exitX=1;exitY=0.5;exitDx=0;exitDy=0;exitPerimeter=0;entryX=1.019;entryY=0.546;entryDx=0;entryDy=0;entryPerimeter=0;" parent="1" source="UDCp61fQdgIPTz9MCS0r-21" target="UDCp61fQdgIPTz9MCS0r-5" edge="1">
           <mxGeometry relative="1" as="geometry" />
         </mxCell>
-        <mxCell id="GbtCg3jwWPSh3iAjcVzU-4" value="! halted" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" parent="GbtCg3jwWPSh3iAjcVzU-3" vertex="1" connectable="0">
+        <mxCell id="GbtCg3jwWPSh3iAjcVzU-4" value="!sec_check" style="edgeLabel;html=1;align=center;verticalAlign=middle;resizable=0;points=[];" parent="GbtCg3jwWPSh3iAjcVzU-3" vertex="1" connectable="0">
```

Comment:
I am aware that the figure is to visualize the specification. It is clear how sec_check behaves in the text section. But, it is not clear in the figure: sec_check[i] = sec_check[i] AND X (What is the signal coming from Hart Trace egress). 

Secondly, it would be better to name sec_check as sec_check[i]

Reply:
Sorry, what do you mean by "sec_check[i] AND X"? I don't get the part AND X. Do you mean halted signal?

>Secondly, it would be better to name sec_check as sec_check[i]

Ack, will update it

Reply:
Thanks! I see the confusion here, let me update the diagram.

Reply:
![external_debug_trace](https://github.com/riscv-non-isa/riscv-external-debug-security/assets/21162377/51c89c3a-0563-4b90-b7df-0377ec7ed9e0)

rework the diagram. @gokhankaplayan could you help review it?

Reply:
Thanks. The updated diagram does make sense to me. How to drive sec_check is already explained in the text section of this Spec. The updated diagram is aligned with the text section. We can resolve this issue.